avm/utilities/tools/Set-AVMModule.ps1

```diff
@@ -76,21 +76,19 @@ function Set-AVMModule {
         [string] $ReadMeScriptFilePath = (Join-Path (Get-Item $PSScriptRoot).Parent.FullName 'pipelines' 'sharedScripts' 'Set-ModuleReadMe.ps1')
     )
 
-    # Load helper scripts
-    . (Join-Path $PSScriptRoot 'helper' 'Set-ModuleFileAndFolderSetup.ps1')
```

Comment:
Is it intended to be removed? This change was introduced recently, I wonder if we missed a pull from latest

Reply:
Hell no. Damn :D

Reply:
Undone